companies/516741394.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "FeelThere",
+  "description": "FeelThere is a live streaming platform that enables simultaneous broadcasting to multiple social networks like Facebook, Instagram, LinkedIn and YouTube. It features a user-friendly interface, high-quality streaming, advanced analytics, and custom watermarks.",
+  "categoryId": 8,
+  "size": "s",
+  "logoFile": "logos/1_FeelThere logo",
+  "websiteUrl": "https://feelthere.live/",
+  "careersUrl": "https://www.linkedin.com/company/feelthere/",
```

Comment:
if there's no careers page on the website, omit this. if you're hiring, put true below.